server_admin/topics/clients/client-policies.adoc

```diff
@@ -0,0 +1,159 @@
+
+Validation of client configurations::
+    The {project_name} supports validation whether the client follows settings like Proof Key for Code Exchange,
+    Request Object Signing Algorithm, Holder-of-Key Token, etc on some endpoints like Authorization Endpoint, Token Endpoint, etc.
+    These can be specified by each setting item (on Admin Console, switch, pulldown menu, etc). To make the client application secure, administrator needs to set
+    lot of such settings in appropriate way, which makes it difficult for the administrator to secure their client application.
+    Client Policies can do these validation of client configurations mentioned just above and they can also be used to auto-configure some client configuration switches to meet
+    the advanced security requirements. In the future, individual client configuration settings may be replaced by Client Policies directly performing required validations.
+
+Conformance to a required security standards and profiles such as FAPI::
+    The _Global client profiles_ are client profiles pre-configured in {project_name} by default. They are pre-configured to be compliant with standard security profiles like link:{adapterguide_link}#_fapi-support[FAPI],
+    which makes it easy for the administrator to secure their client application to be compliant with the particular security profile. At this moment, {project_name} has global
+    profiles for the support of FAPI 1 specification. Administrator will just need to configure the client policies to specify which clients should
+    be compliant with the FAPI. Administrator can configure his own client profiles and client policies, so that {project_name} clients can be easily made compliant with various other
+    security profiles like SPA, Native App, Open Banking etc.
+
+==== Protocol
+
+The client policy concept is independent of any specific protocol. However, {project_name} currently supports it in fact just for the link:{adapterguide_link}#_oidc[OpenID Connect (OIDC) protocol].
+
+==== Architecture
+
+Client Policies consists of the four building blocks: Condition, Executor, Profile and Policy.
+
+===== Condition
+
+A condition determines to which client a policy is adopted and when it is adopted. Some conditions are checked at the time of client create/update when some other conditions are
+checked during client requests (OIDC Authorization request, Token endpoint request etc). The condition checks whether one specified criteria is satisfied. For example,
+some condition checks whether the access type of the client is confidential.
+
+The condition can not be used solely by itself. It can be used in a <<_client_policy_policy,policy>> that is described afterwards.
+
+A condition has it's own SPI. Developers can implement their own condition by implementing the particular provider, however SPI is considered private and can change in the future.
```

Comment:
Should we even mention the SPI if it's private?

Reply:
That is good point. I had an impression, that it may be useful to mention this. But on the other hand, we don't mention this for other SPIs as well. And "geek developers" will figure it anyway without mention in the docs :) So I am removing this.